.github/containerscan/allowedlist.yaml

```diff
@@ -1,6 +1,13 @@
 general:
   vulnerabilities:
     #besu-all-in-one
+    - CVE-2022-37734
+    - CVE-2022-25857
 
-    -CVE-2022-37734
-    -CVE-2022-25857
+    #quorum-multi-party-all-in-one
+    - CVE-2022-24713
+    - CVE-2021-41092
+    - CVE-2022-27664
+    - CVE-2022-32149
+    - CVE-2021-41092
+    - CVE-2022-32149
```

Comment:
@aldousalvarez Same here as with the other quorum image PR: I would remove these from the ignored CVEs for now unless we have a strong reason to not to.

Reply:
Noted on that. Thank you, already fixed it

Reply:
@aldousalvarez Great, thanks for the updates!